### PR TITLE
feat(comment-property): allow column level comments (#4386)

### PR DESCRIPTION
**Pull request by mirkojotic**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](../CONTRIBUTING.md)?

### Description of change
Re-introducing column level comments for mysql, postgres and mssql (sqlite doesn't support comments) #4386

### CODE REVIEW

**Comment by mirkojotic** [comment]
Remove code from another commit ( sorry about that ). Not sure what to do with mssql test suite failing. It doesn't seem like I've changed at that much and the error seems general. Would appreciate any and all feedback and suggestions.

**Comment by mirkojotic** [comment]
@sushantdhiman I've added integration tests for mysql and postgres. In order to check for comment field I had to change `describeTable` statements for both dialects. I can change the mssql statement as well but my tests for mssql integration tests are all failing in docker environment and I've already spent some time exploring why. Since I'm running Ubuntu on my machine I was experiencing difficulties installing it locally.

I will work on documentation.

File: lib/dialects/abstract/query-generator.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ class QueryGenerator {
       })
     );
 
-    return `DESCRIBE ${table};`;
+    return `SHOW FULL COLUMNS FROM ${table};`;
```
Comment (sushantdhiman) [inline lib/dialects/abstract/query-generator.js]: For MySQL? override this method in mysql/query-generator instead

**Comment by sushantdhiman** [comment]
> Since I'm running Ubuntu on my machine I was experiencing difficulties installing it locally.

If you are using docker, you can just

```bash
$ cd /to/sequelize/dev/folder
$ docker-compose up -d mssql

# I recommend .only to run only partial test suite as MSSQL image on Ubnutu fails for transactions
$ DIALECT=mssql npm run test-docker-integration
```

**Comment by mirkojotic** [comment]
Still needs a bit more work. I'll try to finish it up later today.

**Comment by mirkojotic** [comment]
@sushantdhiman hopefully this will do. Let me know if I need to make any corrections. Thank you for all the help and feedback!

**Comment by theRichu** [comment]
In migration process, addColumn and comment is not working

Postgres 10, 
```
        return queryInterface.addColumn(
          {
            schema: 'project',
            tableName: 'scheduleAssigns'
          },
          'timeLogger',
          {
            type: Sequelize.JSONB,
            comment: 'Time Log on JSON',
            defaultValue: {}
          }
        )
```
Generated Query: 
```
ALTER TABLE "project"."scheduleAssigns" ADD COLUMN "timeLogger" JSONB DEFAULT '{}' COMMENT Time Log on JSON;
```

I've got ERROR: syntax error at or near "COMMENT"

Should Be:
```
ALTER TABLE "project"."scheduleAssigns" ADD COLUMN "timeLogger" JSONB DEFAULT '{}'; COMMENT ON "project"."scheduleAssigns"."timeLogger" IS 'Time Log on JSON';
```

**Comment by mirkojotic** [comment]
@theRichu PR you are referencing only adds comments as a part of create table statement. It will not work when modifying table definition. Could be added as a separate issue @sushantdhiman

**Comment by sushantdhiman** [comment]
Now that @theRichu has opened a new PR https://github.com/sequelize/sequelize/pull/10076 we can discuss there

**Pull request merged**